samples/test/support/management-api/deletePolicy.ts

```diff
@@ -0,0 +1,41 @@
+import { Client, AuthorizationServerPolicy } from '@okta/okta-sdk-nodejs';
```

Comment:
Dynamic policy creation is an attempt to avoid coupling tests to a specific org (e.g. for scenarios when QA run tests against different orgs). In case this not very frequent activity(not worth supporting) we can definitely use predefined policy - this will reduce test setup overhead.

Code triggered in `After` step:
https://github.com/okta/okta-auth-js/blob/ced0425c82c753d394e9192ce875e68022f9b6f0/samples/test/support/management-api/deletePolicy.ts#L28

will assign current application to default policy - so in case custom policy is not deleted, next test should still run on default policy. Policy assignment during test run might be problematic when tests are run  in parallel - ideally, we could create a new OIDC app and assign custom policy manually.